connector/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchSourceSuite.scala

```diff
@@ -338,6 +338,7 @@ abstract class KafkaMicroBatchSourceSuiteBase extends KafkaSourceSuiteBase {
     )
 
     // When Trigger.Once() is used, the read limit should be ignored
+    // NOTE: the test uses the deprecated Trigger.Once() by intention, do not change.
```

Comment:
hmm... should we collect them in a new `DeprecatedXXSuite`, which may be easier to notice than comments

Reply:
Probably a good idea for SQL core module but it does not work with connectors. I'm thinking of another mode of Trigger.AvailableNow to behave like Trigger.Once for processing available data (deal with one batch ignoring source options), and then these usages can be replaced with new mode.